### ISSUE TITLE
Can not detect forEach on document.querySelectorAll

### ISSUE THREAD

**Issue by alexander-schranz** [amilajack/eslint-plugin-compat#274]
The following is not detected:

```js
document.querySelectorAll('.sy-container-login-form').forEach((container) => {});
```

This is not detected with the following browserslist:

```
> 1% in alt-EU
last 3 version and not dead
```

Output:

```
and_chr 78
and_ff 68
and_qq 1.2
and_uc 12.12
android 76
baidu 7.12
chrome 78
chrome 77
chrome 76
edge 18
edge 17
edge 16
firefox 70
firefox 69
firefox 68
ie 11
ie_mob 11
ios_saf 13.0-13.2
ios_saf 12.2-12.4
ios_saf 12.0-12.1
kaios 2.5
op_mini all
op_mob 46
opera 64
opera 63
opera 62
safari 13
safari 12.1
safari 12
samsung 10.1
samsung 9.2
samsung 8.2
```

It sjould fail here as IE11 does not support foreach: https://caniuse.com/#feat=mdn-api_nodelist_foreach
